dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -303,13 +182,18 @@ struct metadata_t {
 
     // Flow data
     conntrack_data_t conntrack_data;
+    flow_table_data_t flow_table;
+    dash_flow_state_t flow_state;
```

Comment:
is this duplicated w/ the one in ha_data_t?

Reply:
It is NO by now. Flow state machine and ha state machine  are separated. We may refine it later.

Reply:
this design is weird, what's their difference? if they are not the same, can we split them into 2 types?

Reply:
if they are the same, then we should merge them into one.

Reply:
Merged